src/routes/postSkipSegments.js

```diff
@@ -219,20 +289,24 @@ module.exports = async function postSkipSegments(req, res) {
             return;
         }
 
-        let autoModerateResult = await autoModerateSubmission({videoID, startTime, endTime});
-        if (autoModerateResult) {
-            res.status(403).send("Request rejected by auto moderator: " + autoModerateResult);
-            return;
+        // Auto moderator check
+        if (!isVIP) {
+            let autoModerateResult = await autoModerateSubmission({userID, videoID, startTime, endTime, category: segments[i].category});
+            if (autoModerateResult == "NB disagreement."){
+                // If NB automod rejects, the submission will start with -2 votes
+                //decreaseVotes = -2; //Disable for now
+            } else if (autoModerateResult) {
+                //Normal automod behavior
+                res.status(403).send("Request rejected by auto moderator: " + autoModerateResult + " If this is an issue, send a message on Discord.");
+                return;
+            }
         }
```

Comment:
I suggest we move this out of the for loop. Currently, each segment makes it's own API call to both YT and NB. Moving the loop into `autoModerateSubmission` can reduce it to a single YT and NB API call.

Reply:
Yep, I agree. That makes sense.